index.html

```diff
@@ -0,0 +1,931 @@
+                        <span class="mostPopular">MOST POPULAR</span>
+                        <br/>
+                        <a href="#" class="noTextDecoration popularArticleImageText">
+                            Facebook Gaming launches Tournaments as part of its social distancing campaign
+                        </a>
+                        <br/>
+                        <span class="greyScale"> <a href="#" class="authorNamePopularArticle noTextDecoration">Rachel Kaser</a> 22 hours ago </span>
+                    </div>
+                </div>
+                <div class="latestNewsDescription">
+                    <div class="addPadding">
+                    LATEST
+                    <br/>
+                    <a href="#" class=" fontSizeAndWeight noTextDecoration">
+                        How to disinfect your laptop screen and keyboard
+                    </a>
+                    </div>
+                    <div>
+                        <img class="smallImage" src="https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/03/Person-using-laptop-on-plane-Wi-Fi-hed.jpg" alt="Latest-News-8"/>
+                    </div>
+                </div>
+                <div class="articleLineItems">
+                    <ul>
+                        <li class="borderBottom">
+                            <div><a href="#" class="lineItemStyle noTextDecoration"> A Cats butthole cut ‘trailer’ is out — an analysis of its alternative reality </a></div>
+                        </li>
+                        <li class="borderBottom">
+                            <div><a href="#" class="lineItemStyle noTextDecoration"> Facebook offers $100 million to help news outlets during coronavirus </a></div>
+                        </li>
+                        <li class="borderBottom">
+                            <div><a href="#" class="lineItemStyle noTextDecoration">  Meet the quarantined street photographers documenting life in 'Red Dead Redemption 2' </a></div>
+                        </li>
+                        <li class="borderBottom">
+                            <div><a href="#" class="lineItemStyle noTextDecoration"> The Resident Evil 3 remake falls short of its amazing predecessor </a></div>
+                        </li>
+                        <li class="borderBottom">
+                            <div><a href="#" class="lineItemStyle noTextDecoration"> Here are some mobile and browser games you can play online with friends </a></div>
+                        </li>
+                    </ul>
+                </div>      
+            </article>       
+        </section>
+        <section class="latestDeals">
+            <h2 class="switch">display none</h2>
+            <div class="sectionTitle">
+                <a href="#" class="redHeading noTextDecoration latestDealHeader">
+                    Latest Deals >
+                </a>
+                <a href="#" class="redHeading noTextDecoration seeAll">
```

Comment:
The images in this section have the wrong height. Please adjust it.
Also, check the width of the whole section related to the page.
Yours:
![Screenshot from 2020-04-13 13-32-28](https://user-images.githubusercontent.com/51926937/79148597-59476a00-7d8b-11ea-8df5-d8f1cffd5af1.png)

Original:
![Screenshot from 2020-04-13 13-32-34](https://user-images.githubusercontent.com/51926937/79148610-5e0c1e00-7d8b-11ea-8b9b-284d8e71d251.png)